com.unity.multiplayer.mlapi/Runtime/Core/NetworkManager.cs

```diff
@@ -35,8 +35,16 @@ public class NetworkManager : MonoBehaviour, INetworkUpdateSystem, IProfilableTr
 #if UNITY_2020_2_OR_NEWER
         // RuntimeAccessModifiersILPP will make this `public`
         internal static readonly Dictionary<uint, Action<NetworkBehaviour, NetworkSerializer, __RpcParams>> __ntable = new Dictionary<uint, Action<NetworkBehaviour, NetworkSerializer, __RpcParams>>();
+
+#if UNITY_EDITOR || DEVELOPMENT_BUILD
         // RuntimeAccessModifiersILPP will make this `public`
         internal static readonly Dictionary<uint, string> __rpc_name_table = new Dictionary<uint, string>();
+#else
+        // RuntimeAccessModifiersILPP will make this `public`
```

Comment:
in fact, this won't be processed by `RuntimeAccessModifiersILPP` according to the changes in `RuntimeAccessModifiersILPP.cs`.
I think we should either find a way to avoid this (hardcoding string name of this field might not be the best) or remove this comment line to reflect what's implemented (sounds like we'd maintain some strings manually either way :P, it's just comment vs field name).

Reply:
I think we should keep this as is, but update the comment to better reflect what's happening. I will do that.